src/Modal/Modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode, FC } from 'react'
+import styled, { css } from 'styled-components'
+import { theme } from '../theme'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { Text } from '../Text'
+
+interface IModal {
+  // showModal state
+  showModal: boolean
+}
+
+type ModalProps = {
+  title?: string,
+  body?: string,
```

Comment:
Where is the body used in the component? Am I being dumb 😆 I can't see it.

Reply:
Oh no! Forgot to remove, me being dumb!